platforms/jvm/java-platform/src/test/java/org/gradle/api/plugins/jvm/internal/DefaultJvmPluginServicesTest.groovy

```diff
@@ -74,25 +75,13 @@ class DefaultJvmPluginServicesTest extends AbstractJvmPluginServicesTest {
 
         then:
         1 * attrs.getAttributes() >> mutable
-        0 * _
+        0 * Specification._
```

Comment:
Why are these changing? I don't think we should add this

Reply:
This was automatically done by IDEA, will revert.